src/SFA.DAS.Forecasting.Payments.Functions/PaymentEventHttpFunction.cs

```diff
@@ -25,7 +26,7 @@ public class PaymentEventHttpFunction : IFunction
                     var body = await req.Content.ReadAsStringAsync();
                     var paymentEvent = JsonConvert.DeserializeObject<PaymentCreatedMessage>(body);
 
-                    logger.Info($"Added one payment to {QueueNames.PaymentProcessor} queue.");
+                    logger.Info($"Added one payment to {QueueNames.PaymentProcessor} queue. {paymentEvent.ToJson()}");
```

Comment:
Do we want all the Json in the info log? or should we have a debug one for that

Reply:
Nah, we shouldn't be logging the whole payload as it's a security risk. We should log the payment id, employer accoun, sender account id & funding source maybe.